src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -292,32 +285,49 @@ protected void wrapUp(GHPullRequestCommitDetail[] page) {
+                                            GHPullRequestReviewComment... comments) throws IOException {
+        return createReview(commitId, body, event, Arrays.asList(comments));
+    }
+
+    public GHPullRequestReview createReview(@Nullable String commitId, String body, GHPullRequestReviewEvent event,
+                                            List<GHPullRequestReviewComment> comments) throws IOException {
+        List<DraftReviewComment> draftComments = toDraftReviewComments(comments);
+        return new Requester(root).method("POST")
+                .with("commit_id", commitId)
+                .with("body", body)
+                .with("event", event.action())
+                ._with("comments", draftComments)
+                .to(getApiRoute() + "/reviews", GHPullRequestReview.class)
+                .wrapUp(this);
+    }
+
+    public GHPullRequestReviewDraft newDraftReview(@Nullable String commitId, String body, GHPullRequestReviewComment... comments)
             throws IOException {
-        return createReview(body, event, Arrays.asList(comments));
+        return newDraftReview(commitId, body, Arrays.asList(comments));
     }
 
-    @Preview
-    @Deprecated
-    public GHPullRequestReview createReview(String body, @CheckForNull GHPullRequestReviewState event,
-                                            List<GHPullRequestReviewComment> comments)
+    public GHPullRequestReviewDraft newDraftReview(@Nullable String commitId, String body, List<GHPullRequestReviewComment> comments)
             throws IOException {
-//        if (event == null) {
-//            event = GHPullRequestReviewState.PENDING;
-//        }
-        List<DraftReviewComment> draftComments = new ArrayList<DraftReviewComment>(comments.size());
-        for (GHPullRequestReviewComment c : comments) {
-            draftComments.add(new DraftReviewComment(c.getBody(), c.getPath(), c.getPosition()));
-        }
+        List<DraftReviewComment> draftComments = toDraftReviewComments(comments);
         return new Requester(root).method("POST")
+                .with("commit_id", commitId)
                 .with("body", body)
-                //.with("event", event.name())
                 ._with("comments", draftComments)
-                .withPreview(BLACK_CAT)
-                .to(getApiRoute() + "/reviews", GHPullRequestReview.class).wrapUp(this);
+                .to(getApiRoute() + "/reviews", GHPullRequestReviewDraft.class)
+                .wrapUp(this);
+    }
+
+    private static List<DraftReviewComment> toDraftReviewComments(List<GHPullRequestReviewComment> comments) {
+        List<DraftReviewComment> draftComments = new ArrayList<DraftReviewComment>(comments.size());
+        for (GHPullRequestReviewComment c : comments) {
+            Integer position = c.getPosition();
+            if (position == null) {
+                throw new IllegalArgumentException("GHPullRequestReviewComment must have a position");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 9. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)